pkg/server/test/property_value_aggregate_test.go

```diff
@@ -34,8 +35,91 @@ func Test_property_value_aggregate_query_by_asset_id_and_property_id(t *testing.
 				*entries.PropertyId == "11propid-aaaa-2222-bbbb-3333cccc4444" &&
 				*entries.AggregateTypes[0] == "SUM"
 		}),
+		1,
```

Comment:
It might make sense to do this old test and the new one as [table tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests), since they're almost the same and very wordy.

Reply:
Just wanted to mention that though the individual tests are wordy, table tests come with their own disadvantage that they create a lot of jumping around to read a test from top to bottom "as a story."

After growing a test and copying/pasting, this can create a situation where the test input is hundreds of lines away from the function call under test and assertions.

Reply:
That's fair. I would still lean toward table tests on these, just because these tests specifically have a lot of boilerplate due to the frames that makes it kind of hard to parse what's actually being tested.

Reply:
will take a look at table tests for these today